sdk/containerregistry/azure-containerregistry/samples/sample_base.py

```diff
@@ -0,0 +1,51 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_base.py
+
+DESCRIPTION:
+    This is the base class for sync samples. It imports some images that could be used in samples.
+"""
+import os
+from dotenv import find_dotenv, load_dotenv
+from azure.containerregistry._helpers import _import_image, _get_authority,_get_audience, _get_credential
+
+class SampleBase(object):
```

Comment:
We shouldn't use a base class for samples - it makes it very awkward to read and copy+paste code from the samples